experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -798,6 +798,68 @@ describe("SharedTree", () => {
 			unsubscribe1();
 			unsubscribe2();
 		});
+
+		/**
+		 * the collab window includes all sequenced edits after the minimum sequence number
+		 * these tests test that undoing edits beind the minimum sequence number works
+		 */
+		describe("out of collab window", () => {
+			for (const opNumber of [20, 40, 100]) {
+				it(`an insert ${opNumber} edits behind`, () => {
```

Comment:
Testing that we can undo an insert doesn't leverage repair data.
I think we probably want to test undoing two kinds of edits:
* A deletion
* A change (doesn't matter what) under a previously deleted node